doc/source/getting_started.rst

```diff
@@ -0,0 +1,369 @@
+
+.. ipython:: python
+
+    # array summary : dimensions + description of axes
+    arr.info
+
+    # number of dimensions
+    arr.ndim
+
+    # array dimensions
+    arr.shape
+
+    # number of elements
+    arr.size
+
+    # size in memory
+    arr.memory_used
+
+    # type of the data of the array
+    arr.dtype
+
+Arrays can be generated through dedicated functions:
+
+* :py:func:`zeros` : fills an array with 0
+* :py:func:`ones` : fills an array with 1
+* :py:func:`full` : fills an array with a given
+* :py:func:`eye` : identity matrix
+* :py:func:`ndrange` : fills an array with increasing numbers (mostly for testing)
+* :py:func:`ndtest` : same as ndrange but with axes generated automatically (for testing)
+* :py:func:`sequence` : creates an array by sequentially applying modifications to the array along axis.
+
+.. ipython:: python
+
+   zeros([age_category, sex])
+
+   ndtest((3, 3))
+
+Save/Load an array
+------------------
+
+The LArray library offers many I/O functions to read and write arrays in various formats
+(CSV, Excel, HDF5, pickle). For example, to save an array in a CSV file, call the method
+:py:meth:`~LArray.to_csv`:
+
+.. ipython:: python
+
+    arr_3D = ndtest((2, 2, 2))
+    arr_3D
+
+    arr_3D.to_csv('arr_3D.csv')
+
+Content of 'arr_3D.csv' file is ::
+
+  a,b\c,c0,c1
+  a0,b0,0,1
+  a0,b1,2,3
+  a1,b0,4,5
+  a1,b1,6,7
+
+.. note::
+   In CSV or Excel files, the last dimension is horizontal and the names of the
+   two last dimensions are separated by a \\.
+
+To load a saved array, call the function :py:meth:`read_csv`:
+
+.. ipython:: python
+
+    arr_3D = read_csv('arr_3D.csv')
+    arr_3D
+
+Other input/output functions are described in the :ref:`corresponding section <api-IO>`
+of the API documentation.
+
+Indexing
+--------
+
+To select an element or a subset of an array, use brackets [ ].
+Let’s start by selecting a single element:
+
+.. ipython:: python
+
+    arr = ndtest((4, 4))
+    arr
+
+    arr['a0', 'b1']
+
+    # labels can be given in arbitrary order
+    arr['b1', 'a0']
+
+Let's continue with subsets:
+
+.. ipython:: python
+
+    # select subset along one axis
+    arr['a0']
+
+    arr['b0']
+
+    # labels associated with the same axis must be given as a list
+    # equivalent to: arr['a0', 'b1,b3']
+    arr['a0', ['b1', 'b3']]
+
+.. warning::
+
+    Selecting by labels as above only works as long as there is no ambiguity.
+    When several axes have common labels and you do not specify explicitly
+    on which axis to work, it fails with an error
+    (ValueError: ... is ambiguous (valid in a, b)).
+    Specifying the axis can be done using the special notation ``x.axis_name``.
+    The axis name must not contain whitespaces and special characters.
+
+.. ipython:: python
+
+    # equivalent to: arr2 = ndrange("a=label0,label1;b=label1,label2")
+    arr2 = ndrange([Axis(["label0", "label1"], "a"), Axis(["label1", "label2"], "b")])
+    arr2
+
+    arr2["label0", x.b["label1"]]
```

Comment:
add `# equivalent to: arr2["label0", "b[label1]"]` even though it does not work in master (but it will work before the release)